tests/unit/test-cmd/test.build.js

```diff
@@ -320,6 +322,42 @@ describe('build', () => {
     }
   ));
 
+  it('raises an UsageError if zip file exists', () => {
+    return withTempDir(
+      (tmpDir) => {
+        const testFileName = path.join(tmpDir.path(),
+                                       'minimal_extension-1.0.zip');
+        return fs.writeFile(testFileName, 'test')
+          .then(() => build(
+            {
+              sourceDir: fixturePath('minimal-web-ext'),
+              artifactsDir: tmpDir.path(),
+            }))
+          .catch ((error) => {
+            assert.instanceOf(error, UsageError);
```

Comment:
In addition to asserting the error type, this should also match something specific about the exception message because there are a lot of possible `UsageError`s that can be raised. It doesn't have to match the full message. For example:
````js
assert.match(error.message, /Extension exists at the destination path/);
````